frontend/src/pages/MapView/index.jsx

```diff
@@ -114,8 +48,14 @@ class MapView extends Component {
     </Marker>
   );
 
-  searchHandler = () => {
-    this.setState({ searchResults: searchResults, showResults: true });
+  searchHandler = async () => {
+    const searchResults = await api.getSearchResults(this.state.inputValue);
+    this.setState({
+      searchResults: searchResults,
```

Comment:
Nit, but good tip: Whenever you name something the same in JavaScript as a variable, you can omit the name and just pass `searchResults` instead of `searchResults: searchResults`